lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -1,40 +1,70 @@
 {
-    "workflows": [
+  "workflows": [
+    {
+      "name": "one_step_mediated_deposit",
+      "label": "One-step mediated deposit workflow",
+      "description": "A single-step workflow for mediated deposit",
+      "actions": [
         {
-            "name": "one_step_mediated_deposit",
```

Comment:
The issues were seeking a "depositor review" concept be added to the single step mediated deposit.. so I sketched out an approach to the generator that builds the workflow..